tests/functional/aws-node-sdk/test/object/corsHeaders.js

```diff
@@ -0,0 +1,756 @@
+                    Origin: allowedOrigin,
+                    Test: 'test',
+                    Expires: 86400,
+                };
+                const headersResponse = {
+                    'access-control-allow-origin': allowedOrigin,
+                    'access-control-allow-methods': 'GET',
+                    'access-control-allow-credentials': 'true',
+                    vary,
+                };
+                methodRequest({ method: 'GET', bucket, headers, headersResponse,
+                code: 200 }, done);
+            });
+        });
+
+        describe('cors configuration : MaxAgeSeconds', () => {
+            const corsParams = {
+                Bucket: bucket,
+                CORSConfiguration: {
+                    CORSRules: [
+                        {
+                            AllowedMethods: [
+                                'GET',
+                            ],
+                            AllowedOrigins: [
+                                allowedOrigin,
+                            ],
+                            MaxAgeSeconds: 86400,
+                        },
+                    ],
+                },
+            };
+            beforeEach(done => {
+                s3.putBucketCors(corsParams, done);
+            });
+
+            afterEach(done => {
+                s3.deleteBucketCors({ Bucket: bucket }, done);
+            });
+
+            it('should respond with access-control-max-age header ' +
+            'if request matches CORS rule', done => {
+                const headers = { Origin: allowedOrigin };
+                const headersResponse = {
+                    'access-control-allow-origin': allowedOrigin,
+                    'access-control-allow-methods': 'GET',
+                    'access-control-allow-credentials': 'true',
+                    'access-control-max-age': '86400',
+                    vary,
+                };
+                methodRequest({ method: 'GET', bucket, headers, headersResponse,
+                code: 200 }, done);
+            });
+        });
+
+        describe('cors configuration : ExposeHeaders', () => {
+            const corsParams = {
+                Bucket: bucket,
+                CORSConfiguration: {
+                    CORSRules: [
+                        {
+                            AllowedMethods: [
+                                'GET',
+                            ],
+                            AllowedOrigins: [
+                                allowedOrigin,
+                            ],
+                            ExposeHeaders: [
+                                'Content-Type',
+                            ],
+                        },
+                    ],
+                },
+            };
+            beforeEach(done => {
+                s3.putBucketCors(corsParams, done);
+            });
+
+            afterEach(done => {
+                s3.deleteBucketCors({ Bucket: bucket }, done);
+            });
```

Comment:
helper function for corsParams: https://github.com/scality/S3/pull/541/commits/3aa0e7e09ab3c3fcbc5c58855b104e3deb9ad695?diff=split
forEach for last two tests:
https://github.com/scality/S3/pull/541/commits/81b3d0e328bd6276d862d831de2e38e21431670e?diff=split